scripts/analysis.py

```diff
@@ -720,9 +825,11 @@ def stacked_bar_chart(dictionary, timestep,
         if isinstance(key, str) is True:
             label = key.replace('_government', '')
         else:
-            label = str(nucname.name(key))
+            label = str(key)
         # very first country does not have a 'bottom' argument
-        if top_index is True:
+        if "Sink" in label:
+            print("Ignore Sink Institution")
```

Comment:
not very generic behavior. ok for now, I suppose, but it will undermine future analyses.